ingestion/src/metadata/ingestion/source/dashboard/metabase.py

```diff
@@ -14,6 +14,7 @@
 from typing import Iterable, List, Optional
 
 import requests
+from sqllineage.runner import LineageRunner
```

Comment:
Could we update the import bypassing the conf? Otherwise we had issues with our logging system https://github.com/open-metadata/OpenMetadata/blob/main/ingestion/src/metadata/ingestion/source/dashboard/mode.py#L44

Reply:
Done, could you take a look at it again?